client/extensions/woocommerce/state/wc-api/settings/general/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL,
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL_SUCCESS,
+} from '../../../action-types';
+
+export const LOADING = 'LOADING';
```

Comment:
I'm not a big fan of this either. I feel this is metadata and doesn't really belong in the state data. But we don't have a really good place to put information like this yet either.